async/pool_test.ts

```diff
@@ -22,7 +22,7 @@ Deno.test("pooledMap()", async function () {
   assert(diff < 3000);
 });
 
-Deno.test("pooledMap() handles errors", async () => {
+Deno.test("pooledMap() handles errors", async function () {
```

Comment:
Nit: we generally use the arrow syntax for anonymous functions. Please revert. Ditto elsewhere.

Reply:
understood! also took liberty to convert the other test cases in the async sub-module to make use of the arrow syntax as well.